tests/Pipelines/Stages/ReIndexStageTest.php

```diff
@@ -17,8 +17,19 @@ class ReIndexStageTest extends AbstractStageTestCase
     public function testWhenIndexExists()
     {
         $indices       = $this->getMockedIndices(['exists', 'putSettings']);
+        $tasks         = $this->getMockedTasks(['get']);
         $searchService = $this->getMockedSearchService($indices);
 
+
+        $tasks->expects($this->once())
+              ->method('get')
+              ->willReturn(['completed' => true]);
```

Comment:
I think it should be possible to mock it so that it returns `['completed' => false]` on the first call, then `['completed' => true]` on the second one. That way you could check that `tasks()` is called exactly two times.